app/services/art_service/report_engine.rb

```diff
@@ -123,6 +124,10 @@ def tx_ml(start_date, end_date)
       REPORTS['TX_ML'].new(start_date: start_date.to_date, end_date: end_date.to_date).data
     end
 
+    def tx_tb(start_date, end_date)
```

Comment:
This is not necessary as long as the find_report method is defined on the TxTB class